### DISCUSSION TITLE
Bot is spamming my site

### DISCUSSION THREAD

**Discussion by HuyAms** [remix-run/remix#4787] [Q&A]
Hi,

I built a site with Remix and deployed it to Fly. I have a NodeJS server. Actually, I use the same server as the one in the blues-stack
https://github.com/remix-run/blues-stack/blob/main/server.ts

Recently, a bot spams my server. Someone used bad bots to send a lot of traffic to my website. It keeps requesting our blog pages with a really weird query string.  

What should I do to track and block those bots?

I am thinking of using middleware like [express-rate-limit](https://github.com/express-rate-limit/express-rate-limit) but I don't know what should be the value the `windowMs`  and `max`.

Thanks so much 🙏

**Comment by qianjiachun** [comment]
My problem is that all `request.ip` come from 127.0.0.1😂
Maybe need to write some customized rules?
